mkdocs.yml

```diff
@@ -24,14 +24,16 @@ extra:
       short_name: "CC BY-SA"
       url: "https://creativecommons.org/licenses/by-sa/4.0/"
     cdpa:
+      date: 2023-10-02
```

Comment:
@armanborghem: Please let us know if this date should be replaced with the date of publication (that is, 2023-10-03).

Reply:
Here: <https://docs.cleura.cloud/reference/legal/#data-processing-agreement>

Reply:
In my view, the update date indicated on the docs pages should be the actual date each page is updated.

Reply:
I think the public documentation should reflect the date as of which the terms *apply,* as that is the date that is of relevance to our customers. So it should say 2023-10-03. I'll update the PR accordingly.

Reply:
I would prefer if the text was changed to:

Our standard Data Processing Agreement for Public Cloud was last updated on 2023-10-03.
(link "Data Processing Agreement for Public Cloud")

Reply:
The most recent version of this PR lists both the date of last update, and the date of applicability. I think that's the best we can do.